Modules/selectmodule.c

```diff
@@ -232,11 +232,11 @@ select.select
 
 Wait until one or more file descriptors are ready for some kind of I/O.
 
-The first three arguments are sequences of file descriptors to be waited for:
+The first three arguments are iterables of file descriptors to be waited for:
 rlist -- wait until ready for reading
 wlist -- wait until ready for writing
 xlist -- wait for an "exceptional condition"
-If only one kind of condition is required, pass [] for the other lists.
```

Comment:
ISTM that in this line, the original "pass []" is clearer and simpler than "pass empty iterables", and is still correct.

Reply:
That's fair, I brought back the original.